ui/job-view/KeyboardShortcuts.jsx

```diff
@@ -66,39 +67,27 @@ class KeyboardShortcuts extends React.Component {
 
   // select next unclassified failure
   nextUnclassified() {
-    this.$rootScope.$emit(
-      thEvents.changeSelection,
-      'next',
-      thJobNavSelectors.UNCLASSIFIED_FAILURES);
+    this.props.changeSelectedJob('next', true);
```

Comment:
Could you get rid of these four separate methods and put the `changeSelectedJob` method right into the handler?

Reply:
Good call, yeah.  :)